airframe-sql/src/main/scala/wvlet/airframe/sql/analyzer/TypeResolver.scala

```diff
@@ -463,7 +463,7 @@ object TypeResolver extends LogSupport {
           case other        => toResolvedAttribute(other.name, other)
         }
         List(a.copy(columns = Some((qualifier match {
-          case Some(q) => allColumns.filter(c => c.qualifier.contains(q) || c.tableAlias.contains(q))
+          case Some(q) => allColumns.filter(_.tableAlias.contains(q))
```

Comment:
Good. This is because AllColumn.outputAttributes have both the qualifier and tableAlias

Reply:
> This is because AllColumn.outputAttributes have both the qualifier and tableAlias

This is not accurate explanation.

Without this fix, we needed to check qualifiers for the following case but now this case is covered by `tableAlias`.

```sql
select t1.* from (select t1.time from A) t1
```

On the other hand, checking qualifiers allows to resolve cases like below, but actually this is not a valid SQL which means the current logic can resolve even invalid cases.

```sql
select A.* from (select A.time from A)
select t1.* from (select t1.time from A t1)
```